app/src/test/java/com/nervousfish/nervousfish/data_objects/RSAKeyTest.java

```diff
@@ -0,0 +1,78 @@
+package com.nervousfish.nervousfish.data_objects;
+
+import com.nervousfish.nervousfish.ConstantKeywords;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+public class RSAKeyTest {
+
+    @Test
+    public void testCanBeInstantiatedWithArbitraryValues() {
+        IKey key = new RSAKey("foo", "bar");
+        assertNotNull(key);
+    }
+
+    @Test
+    public void testGetKeyReturnsNotNull() {
+        IKey key = new RSAKey("foo", "bar");
+        assertNotNull(key.getKey());
+    }
+
+    @Test
+    public void testGetKeyReturnsNonEmptyString() {
+        IKey key = new RSAKey("foo", "bar");
+        assertNotEquals("", key.getKey());
+    }
+
+    @Test
+    public void testGetTypeReturnsRSA() {
+        IKey key = new RSAKey("foo", "bar");
+        assertEquals(ConstantKeywords.RSA_KEY, key.getType());
```

Comment:
Do you think it does not look nice to have more than 1 assert in a test? I think that is better understandable than having duplicate code for each new assert.